internal/pkg/manifest/pipeline.go

```diff
@@ -21,6 +21,12 @@ const (
 	pipelineManifestPath = "cicd/pipeline.yml"
 )
 
+var PipelineProviders = []string{
```

Comment:
Can we add doc comments please? 🙏  https://github.com/golang/go/wiki/CodeReviewComments#doc-comments
```suggestion
// PipelineProviders is the list of all available source integrations.
var PipelineProviders = []string{
```